features/step_definitions/user_steps.rb

```diff
@@ -42,39 +42,44 @@
 Given(/^I am logged into the site$/) do
   visit new_user_session_path
   FactoryGirl.create(:user)
-  fill_in 'Email', with: 'newuser@sb.com' 
-  fill_in 'Password', with: 'password' 
+  fill_in 'Email', with: 'newuser@sb.com'
+  fill_in 'Password', with: 'password'
   click_on 'Log in'
 end
 
 When(/^I visit my own profile page$/) do
-  pending # express the regexp above with the code you wish you had
+ visit root_path
 end
 
 Then(/^I see my username$/) do
-  pending # express the regexp above with the code you wish you had
+  expect(page).to have_content("NewUser")
```

Comment:
This test works as-is, but it's relying on your factory code. So if you ever change the username in your factory, then it will break this test. Ideally, what you wanna do here is get "my username" and compare those. I'm not sure of a good way to grab that though...